easybuild/easyconfigs/b/BamM/BamM-1.7.3-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     GPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'BamM'
+version = '1.7.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ecogenomics.github.io/BamM/'
+description = """ BamM is a c library, wrapped in python, that parses BAM files """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Ecogenomics/BamM/archive']
+sources = ['%(version)s.tar.gz']
+
+checksums = [
+    '1a30a5014aa64aea23f12b82c8e474044af126c9e6ddb575530ab14b6ef765b8',
+]
+
+builddependencies = [
+    ('Automake', '1.11.3'),
```

Comment:
@robqiao yes, I did spot quite a bit of duplication during my failed build attempt yesterday, so I echo the comment by @boegel